frontend/src/actions/parcelsActions.tsx

```diff
@@ -2,18 +2,19 @@ import * as ActionTypes from 'constants/actionTypes';
 
 //Parcel List API action
 
-export interface IParcel {
+export interface IProperty {
   id: number;
+  propertyTypeId: 0 | 1; // 0 = Parcel, 1 = Building
```

Comment:
This can be tech dept, but I'd like to move these to named constants or enum at some point.

Reply:
Agree - we should create a custom `type PropertyType = 0 | 1`. I didn't go with `number` to be explicit about the fact that we only support 0 or 1 at the moment and avoid bugs where the propTypeId is set to "3"